js/asurite-id-field.js

```diff
@@ -3,6 +3,14 @@
 
   Drupal.behaviors.asuriteIdField = {
     attach: function (context, settings) {
+
+      // Check when to display the widget. It is not removed in case we go back
+      // to having this option.
+      check_azurite_widget_display();
```

Comment:
Can we use `asurite` instead of `azurite` in the code for accuracy and consistency?